rest/route/github.go

```diff
@@ -31,13 +33,15 @@ type githubHookApi struct {
 	eventType string
 	msgID     string
 	sc        data.Connector
+	settings  *evergreen.Settings
 }
 
-func makeGithubHooksRoute(sc data.Connector, queue amboy.Queue, secret []byte) gimlet.RouteHandler {
+func makeGithubHooksRoute(sc data.Connector, queue amboy.Queue, secret []byte, settings *evergreen.Settings) gimlet.RouteHandler {
 	return &githubHookApi{
-		sc:     sc,
-		queue:  queue,
-		secret: secret,
+		sc:       sc,
+		settings: settings,
+		queue:    queue,
+		secret:   secret,
```

Comment:
you also need to propogate the settings in the Factory.

Reply:
:cold_sweat:
done.